packages/vuetify/src/composables/hotkey.ts

```diff
@@ -0,0 +1,176 @@
+// Utilities
```

Comment:
If this is a standalone it should be in its own PR

Reply:
Just a duplicate to remove.